nextcloud-startup-script.sh

```diff
@@ -402,7 +403,13 @@ do
             clear
             run_static_script cookielifetime
         ;;
-
+		
+        "Share-Folder")
+            clear
+            occ_command config:system:set share_folder --value="/Shared"
+            msg_box "All new Nextcloud-shares from other users will be stored in the 'Shared' folder by now."
```

Comment:
... users will be stored in the 'Shared' folder from now on

Reply:
done 👍